data/account/participation.go

```diff
@@ -215,6 +216,11 @@ func (part PersistedParticipation) PersistNewParent() error {
 	})
 }
 
+// DefaultKeyDilution computes the default dilution based on first and last rounds.
+func DefaultKeyDilution(first, last basics.Round) uint64 {
+	return 1 + uint64(math.Sqrt(float64(last-first)))
```

Comment:
Looks like the equation has been kicking around for years: https://github.com/algorand/go-algorand/commit/a0f4512bef891162e8180f0abe99288cc2328480#diff-45a0bd848fce2d3f3326e33db0b81ad361d20ea9bd4dad6773513e635885d82aR59

I'll update the other spots.

Reply:
Ok, I think it's wrong, but I'm certainly not going to worry about it.

Reply:
Yes, but why? I can't see that there's any harm done by being wrong by a little bit either way - you get an extra batch if you err in one direction, you get slightly bigger batches if you err in the other direction. So I thought maybe they were just trying to avoid the one actual problem, which would be getting a zero because first==last.

Anyway, I guess my proposal isn't really any better.  I suspect it will always give the same answer with the size numbers we're using. Better not to change anything.

Reply:
Yeah, I just found that too! I thought they both started at 0, in the firstVote round. Nope.

It's nice because you can do all the calculations with knowing firstVote.